testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterBuilder.kt

```diff
@@ -388,6 +388,22 @@ class ClusterBuilder {
         return "{$bodyStr1$bodyStr2}"
     }
 
+    /** Get schema SQL to create crypto DB */
+    fun createCryptoSchemaSql() =
```

Comment:
```suggestion
    fun getCryptoSchemaSql() =
```
`create` may give an impression that we are inserting something into the database or creating some sort of an entity, which is not being the case.